src/service/rpc/generic/LoadBalancer.ts

```diff
@@ -104,11 +104,13 @@ export class LoadBalancer implements AbstractRpcInterface {
       process.on('exit', () => this.destroy())
     }
 
-    if (config.rpc?.oneTimeLoadBalancing) {
-      Utils.log({ id: this.id, message: 'oneTimeLoadBalancing enabled' })
-      await this.checkStatuses()
-    } else {
-      this.interval = setInterval(() => this.checkStatuses(), Constant.OPEN_RPC.LB_INTERVAL)
+    if (nodes && nodes.length > 1) {
+      if (config.rpc?.oneTimeLoadBalancing) {
+        Utils.log({ id: this.id, message: 'oneTimeLoadBalancing enabled' })
+        await this.checkStatuses()
```

Comment:
What is the default health status if checkStatuses() is never ran, will the node still be routable?

Reply:
Initially the node has flag `failed: false` so it is expected to be working.

I have tested it and if you manually enter single node, it will work as expected.